javascripts/discourse/components/custom-categories-navbar.js

```diff
@@ -14,7 +14,13 @@ export default class CustomCategoriesNavbar extends Component {
   }
 
   get shouldRender() {
-    return !this.router.currentRouteName.startsWith("chat");
+    const isChat = this.router.currentRouteName.startsWith("chat");
+    const isMinimized = this.args.outletArgs.minimized;
+    const alwaysHideDocked = settings.hide_on_topic_scroll
+      ? isMinimized
+      : this.site.mobileView && isMinimized;
+
+    return !(alwaysHideDocked && !isChat) && !isChat;
```

Comment:
So this is the same as:

```suggestion
    return (!alwaysHideDocked || isChat) && !isChat;
```

which shows that `|| isChat` part is redundant

```suggestion
    return !alwaysHideDocked && !isChat;
```

Right? 😅